go/vt/vttablet/tabletmanager/rpc_query.go

```diff
@@ -275,6 +275,18 @@ func (tm *TabletManager) ExecuteFetchAsApp(ctx context.Context, req *tabletmanag
 	return sqltypes.ResultToProto3(result), err
 }
 
+// MysqlSystemMetrics gets system metrics from the MySQL deamon
+func (tm *TabletManager) MysqlSystemMetrics(ctx context.Context, req *tabletmanagerdatapb.MysqlSystemMetricsRequest) (*tabletmanagerdatapb.MysqlSystemMetricsResponse, error) {
+	mysqlResp, err := tm.MysqlDaemon.SystemMetrics(ctx, tm.Cnf)
```

Comment:
This is the bridge between `TabletManager` and `MysqlDaemon`. It is fully functional. But right now, no one will be calling this function, so there's no backwards-compatibility issue.